optimum/commands/export/openvino.py

```diff
@@ -293,6 +293,12 @@ def parse_args_openvino(parser: "ArgumentParser"):
         type=json.loads,
         help=("Any kwargs passed to the model forward, or used to customize the export for a given model."),
     )
+    optional_group.add_argument(
+        "--use-torch-export",
```

Comment:
Have you considered the name `--torch-export` for this argument? I think it is more concise without any drawbacks.

Reply:
Agree, renamed.